beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -1,56 +1,56 @@
+आप इस तरह का "कंटेनर" एलिमेंट `<div>` को `<body>` टैग के अंदर कहीं भी रख सकते हैं। React HTML एलिमेंट के अंदर किसी भी मौजूदा कंटेंट को बदल देगा, इसलिए वे आमतौर पर खाली होते हैं। आप अपनी ज़रूरत के हिसाब से ऐसे कितने भी HTML एलिमेंट्स एक पेज पर इस्तेमाल कर सकते हैं।
 
 ```html {3}
-<!-- ... existing HTML ... -->
+<!-- ... मौजूदा HTML ... -->
 
-<div id="component-goes-here"></div>
+<div id="कौम्पोनॅन्ट-यहाँ-जाएगा"></div>
 
-<!-- ... existing HTML ... -->
+<!-- ... मौजूदा HTML ... -->
 ```
 
-### Step 2: Add the Script Tags {/*step-2-add-the-script-tags*/}
+### स्टेप 2: script टैग ऐड करें {/*step-2-add-the-script-tags*/}
 
-In the HTML page, right before the closing `</body>` tag, add three `<script>` tags for the following files:
+HTML पेज में, समापन `</body>` टैग से ठीक पहले, निम्न फ़ाइलों के लिए तीन `<script>` टैग ऐड करें:
 
-- [**react.development.js**](https://unpkg.com/react@17/umd/react.development.js) loads the core of React
-- [**react-dom.development.js**](https://unpkg.com/react-dom@17/umd/react-dom.development.js) lets React render HTML elements to the [DOM](https://developer.mozilla.org/docs/Web/API/Document_Object_Model).
-- **like_button.js** is where you'll write your component in step 3!
+- [**react.development.js**](https://unpkg.com/react@17/umd/react.development.js) React के मूल को लोड करता है
+- [**react-dom.development.js**](https://unpkg.com/react-dom@17/umd/react-dom.development.js) React को HTML एलिमेंट को प्रस्तुत करने देता है [DOM](https://developer.mozilla.org/docs/Web/API/Document_Object_Model).
+- **like_button.js** वह जगह है जहाँ आप स्टेप 3 में अपना कौम्पोनॅन्ट लिखेंगे!
 
 <Gotcha>
 
-When deploying, replace "development.js" with "production.min.js".
+परिनियोजित करते समय, "Development.js" को "production.min.js" से बदलें।
 
 </Gotcha>
 
 ```html
-  <!-- end of the page -->
+  <!-- पेज का अंत -->
   <script src="https://unpkg.com/react@17/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@17/umd/react-dom.development.js" crossorigin></script>
   <script src="like_button.js"></script>
 </body>
 ```
 
-### Step 3: Create a React component {/*step-3-create-a-react-component*/}
+### स्टेप 3: एक React कौम्पोनॅन्ट बनाएँ {/*step-3-create-a-react-component*/}
 
-Create a file called **like_button.js** next to your HTML page, add this code snippet, and save the file. This code defines a React component called `LikeButton`. [You can learn more about making components in our guides.](/learn/your-first-component)
+अपने HTML पेज के बगल में **like_button.js** नाम की एक फाइल बनाएं, इस कोड स्निपेट को ऐड करें और फाइल को सेव करें। यह कोड `LikeButton` नामक एक React कौम्पोनॅन्ट को परिभाषित करता है। [आप हमारे गाइड में कौम्पोनॅन्ट बनाने के बारे में अधिक जान सकते हैं।](/learn/your-first-component)
```

Comment:
`परिभाषित` => `डिफाइन`

Not very sure about this. Do you have any other alternatives here? `परिभाषित` sounds a bit complex.